docs/release-notes/FSharp.Core.fsx

```diff
@@ -1,4 +1,4 @@
-﻿(** ---
+(** ---
```

Comment:
Did you change line ending here or why is this line listed? :>

Reply:
Uh, not sure, I created PR via iOS client.

Reply:
Maybe it stripped bom or something?